Cargo.toml

```diff
@@ -3,10 +3,15 @@ name = "xtask-wasm"
 version = "0.0.0"
 edition = "2021"
 license = "MIT OR Apache-2.0"
-description = "Extend your xtask workflow with custom WASM commands"
+description = "Extend your project workflow with custom xtask command for WASM"
 homepage = "https://github.com/rustminded/xtask-wasm"
 repository = "https://github.com/rustminded/xtask-wasm"
 documentation = "https://docs.rs/xtask-wasm"
+rust-version = "1.58.1"
```

Comment:
Ahh that's how it's done! Don't forget to leave a comment with the link to the security issues. Otherwise in exactly 2 months someone is going to make a PR to remove that line and say "useless restrictions".

Remember that documenting things is very important in our job as things tend to live for much longer than we anticipate sometimes.